test/ipaddress/ipv4_test.rb

```diff
@@ -433,6 +433,57 @@ def test_method_supernet
     assert_equal "172.16.8.0/22", @ip.supernet(22).to_string
   end
 
+  def test_method_add
+    ip = IPAddress::IPv4.new("172.16.10.1/24")
+    assert_equal ip.add(5), IPAddress::IPv4.new("172.16.10.6/24")
```

Comment:
There are no parens in the existing tests, should they be added everywhere? I'm currently integrating your suggestions.